domain/resource/errors/errors.go

```diff
@@ -53,8 +53,4 @@ const (
 	// cannot be found in the relevant resource persistence layer for its
 	// resource type.
 	StoredResourceNotFound = errors.ConstError("stored resource not found")
-
-	// ResourceAlreadyStored describes an errors where the resource has already
```

Comment:
I'm assuming this is safe to remove as the service/state methods are idempotent?

Reply:
Exactly, they are.